Shorten CTA wording for reviewers, voting, and planning

hypha opened the pull request:
Closes https://github.com/EuroPython/website/issues/999
Made them all fit into three lines as well :sweat_smile:

hypha committed:
Shorten CTA wording for reviewers, voting, and planning

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

clytaemnestra commented:
Overall LGTM and thank you very much for the initiative!

> Be part of the reviewer team! YOUR insights help us curate a diverse and high-quality programme.

* I'd consider changing `reviewer team` to `team that reviews talk proposals`, because to a newbie it might not be clear what do they review 
* I'm not sure if the word `curate` is well-known to non-native speakers, maybe `creates`? 

> Share your thoughts and suggestions in our planning survey! Help us build a conference that works for YOU!
* I'd maybe add `... suggestions what is missing at EP` - people are usually quick to criticize and I think we can ride that wave and use their feedback. If it sounds too negative, then maybe `what you would like to see at EP` 
EDIT: I meant the first sentence as `Share your thoughts what is missing at EP.` or alternatively `Share your suggestions what would you like to see at EP`. Also not to repeat `planning survey` in the title + first sentence. 

These are my suggestions and nitpicks, feel free to disregard if it doesn't make sense to you 🙂

hypha committed:
Address review suggestions

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hypha commented:
Thanks for the reviews, @clytaemnestra!
 
>  I'd consider changing `reviewer team` to `team that reviews talk proposals`, because to a newbie it might not be clear what do they review.

Alright! Changed in https://github.com/EuroPython/website/pull/1006/commits/4325f720b4faa672655a94855e9a547f3458b2f6.

> I'm not sure if the word `curate` is well-known to non-native speakers, maybe `creates`?
 
The word `curate` (or a related form of it) exists in many Indo-European languages, so I’m not too concerned about comprehension. But I’ve changed it to `creates` in https://github.com/EuroPython/website/pull/1006/commits/4325f720b4faa672655a94855e9a547f3458b2f6 as suggested, since it also works well.
 

> I'd maybe add `... suggestions what is missing at EP` - people are usually quick to criticize and I think we can ride that wave and use their feedback. If it sounds too negative, then maybe `what you would like to see at EP`
> EDIT: I meant the first sentence as `Share your thoughts what is missing at EP.` or alternatively `Share your suggestions what would you like to see at EP`. Also not to repeat `planning survey` in the title + first sentence.

I’ve reworded it to be slightly more open-ended in https://github.com/EuroPython/website/pull/1006/commits/4325f720b4faa672655a94855e9a547f3458b2f6, as I felt `what’s missing` suggests only new additions, while `what you’d like to see` could steer responses too much toward programme-related suggestions. I’d like to encourage broader feedback, including thoughts on existing elements that could be improved. Let me know what you think. Happy to adjust!

clytaemnestra reviewed (approved):
Looks great, thank you!
